opencog/atoms/execution/CMakeLists.txt

```diff
@@ -56,5 +56,6 @@ ENDIF (HAVE_GUILE)
 # are fixed, first.
 INSTALL (FILES
 	MapLink.h
-	DESTINATION "include/opencog/atoms/core"
+	LibraryManager.h
```

Comment:
I think I had previously removed `LibraryManager.h` from the install list, because it had struck me as a horrid, badly-designed API and I did not want to encourage it's use. Do you actually need it for something?

Reply:
The URE needs it to extract the formula name of a rule https://github.com/opencog/atomspace/blob/4abc384dd366fd36288da6ff84d28f59c1426da6/opencog/ure/backwardchainer/BIT.cc#L707

It's not crucial but it's convenient for pretty printing inference trees.